docs/hints.md

```diff
@@ -0,0 +1,45 @@
+# Hints
+
+Hints let you provide information on failing checks.
+
+![Hint Example](../misc/gh/ReadMe.png)
+
+Hints are a way to help make images more approachable.
+
+You can add a conditional hint or a check-wide hint. A conditional hint is printed when that specific condition is executed and fails. Make sure you understand the check precedence; this can be tricky, as sometimes your check is NOT executed ([read about conditions](conditions.md)).
+
+Example conditional hint:
+```
+[[check]]
+points = 5
+
+	[[check.pass]]
+	type = "ProgramInstalledNot"
+	name = "john"
+
+	[[check.pass]]
+	# This hint will NOT print unless the condition above succeeds.
+	# Pass conditions are logically AND-- they all need to succeed.
+	# If one fails, there's no reason to execute the other ones.
+	hint = "Removing just the binary is insufficient; use a package manager to remove all of a tool's files."
+	type = "PathExistsNot"
+	path = "/usr/share/john"
+```
+
+Check-wide hints are at the top level and always displayed if a check fails. Example check-wide hint:
+
+```
+[[check]]
+hint = "Are there any 'hacking' tools installed?"
+points = 5
+
+	[[check.pass]]
+	type = "ProgramInstalledNot"
+	name = "john"
+
+	[[check.pass]]
+	type = "PathExistsNot"
+	path = "/usr/share/john"
+```
+	
```

Comment:
so you would end up with two hints to the same check? Which can be confusing when its based on points so
(Points: 5) (hint)
(Points: 5) (another hint on the same thing, but it could be interpreted as solving both hints give you 10 points)

Reply:
When points are for the same item they're grouped in a list so I don't think that would be confusing. Maybe vice versa if there are two hints for 5 points someone could think they're the same underlying check but I think that's unlikely